src/components/Main/Card/Card.jsx

```diff
@@ -0,0 +1,31 @@
+import defalutimg from "../../../assets/defalutimg.png";
+import getElapsedTime from "../../../utils/calculateDate.js";
+import "./Card.css";
+
+function Card({ data }) {
+  const { title, description, url, imageSource, createdAt } = data;
+
+  const timeAgo = getElapsedTime(createdAt);
+  const creatDate = createdAt.substring(0, 10).replaceAll(/-/g, ". ");
+
+  return (
+    <li className="card">
+      <a href={url} target="_blank">
+        <div className="card-imgbox">
+          <img src={imageSource || defalutimg} alt={title} />
```

Comment:
```suggestion
          <img src={imageSource ?? defalutimg} alt={title} />
```
[중요도: 중간] 위 방식을 권장 드립니다.

?? 연산자: nullish를 검증 (null or undefined)
|| 연산자: falsy value도 검증한다.

당장에 있어서는 ||도 되지만 명시적인 관점에서는 ??가 맞는 표현이니까요~